test/verify/check-kdump

```diff
@@ -113,8 +113,6 @@ class TestKdump(KdumpHelpers):
         b.set_input_text(serverInput, "")
         b.click(f"#kdump-settings-dialog button{self.primary_btn_class}")
         b.wait_in_text("#kdump-settings-dialog h4.pf-c-alert__title", "Unable to save settings: nfs server is empty")
-        # no further details/journal
-        self.assertFalse(b.is_present("#kdump-settings-dialog .pf-c-code-block__code"))
```

Comment:
This or some replacement should come back when reintroducing pre-wrapped error message details.

Reply:
Right, I have done so for the line below. But as the dialog now does proper validation a user can't even submit the dialog if the server is empty.

Reply:
The commit still removes this without replacement. But the next commit changes that again for the inline validation, which is fine.